boringssl-static/pom.xml

```diff
@@ -1113,6 +1113,254 @@
         </plugins>
       </build>
     </profile>
+
+    <!-- Profile to cross-compile for mac intel -->
+    <profile>
+      <id>mac-intel-cross-compile</id>
+
+      <properties>
+        <linkStatic>true</linkStatic>
+        <osxCrossCompile>true</osxCrossCompile>
+        <target>x86_64-apple-macos10.12</target>
+        <macOsxDeploymentTarget>OSX_DEPLOYMENT_TARGET=10.12</macOsxDeploymentTarget>
+        <cmakeOsxDeploymentTarget>-DCMAKE_OSX_DEPLOYMENT_TARGET=10.12</cmakeOsxDeploymentTarget>
+        <!-- On *nix, add ASM flags to disable executable stack -->
+        <cmakeAsmFlags>-Wa,--noexecstack -target ${target}</cmakeAsmFlags>
+        <!-- Use -DOPENSSL_C11_ATOMIC so we replace most of the locking code with atomics-->
+        <cmakeCFlags>-O3 -fno-omit-frame-pointer -target ${target} -DOPENSSL_C11_ATOMIC</cmakeCFlags>
+        <cmakeCxxFlags>-O3 -fno-omit-frame-pointer -target ${target} -Wno-error=range-loop-analysis</cmakeCxxFlags>
+        <cflags>-target ${target} -Werror -fno-omit-frame-pointer -fvisibility=hidden -Wunused -Wno-unused-value -O3</cflags>
+        <cppflags>-target ${target} -DHAVE_OPENSSL -I${boringsslSourceDir}/include</cppflags>
+        <ldflags>-arch arm64 -L${boringsslHome}/ssl -L${boringsslHome}/crypto -lssl -lcrypto</ldflags>
+        <!-- use aarch_64 as this is also what os.detected.arch will use on an aarch64 system -->
+        <nativeJarFile>${project.build.directory}/${project.build.finalName}-${os.detected.name}-osx_64.jar</nativeJarFile>
+        <nativeLibOsParts>${os.detected.name}_osx_64</nativeLibOsParts>
+        <jniClassifier>${os.detected.name}-osx_64</jniClassifier>
+        <jniArch>osx_64</jniArch>
+        <javaModuleNameClassifier>${os.detected.name}.osx_64</javaModuleNameClassifier>
```

Comment:
What is the `osx_64` part used in these lines? Should it be `osx_x86_64` or something like that?

Reply:
typo